packages/PreviewCard/PreviewCard.jsx

```diff
@@ -143,6 +145,8 @@ PreviewCard.defaultProps = {
   href: null,
   linkComponent: null,
   to: null,
+  image: null,
```

Comment:
Please use `undefined` instead of `null` for default values since `undefined` is more flexible with common JSX patterns such as conditional rendering:

```jsx
<>
  {text && <span>{text}</span>}
</>
```